composer.json

```diff
@@ -16,10 +16,10 @@
         "drupal/console": "~1.0",
         "drupal/devel": "^2.1",
         "drush/drush": "~9",
-        "localgovdrupal/localgov_core": "dev-master",
+        "localgovdrupal/localgov_core": "dev-feature/14-move-block-config",
```

Comment:
Want to remember _not_ to merge this, but to merge all other branches at the same item.